lib/cm_clickpost/payload_helper.rb

```diff
@@ -0,0 +1,113 @@
+module CmClickpost
+    module PayloadHelper
+  
+      class << self
+
+        def pickup_address_payload(pickup_info = {})
+          {
+              "pickup_state": pickup_info.fetch(:pickup_state),
+              "pickup_address": pickup_info.fetch(:pickup_address),
+              "email": pickup_info.fetch(:email),
+              "pickup_time": pickup_info.fetch(:pickup_time_in_iso_format),
```

Comment:
1. Payload example needs to be added in documentation
2. If they are sending the hash why do we need to map it?

@jish0desarkar @mikevic 

To make it easier on the client side, we should look at creating classes for each object. Something like
ClickPost::PickupAddress.new(pickup_state: '', ....)
ClickPost::DropAddress.new(pickup_state: '', ....)

Later when we execute we can turn this into JSON object and make the request. This can be a improvement.

Reply:
I too would prefer using a class object instead and converting it to JSON when needed.